src/networking.c

```diff
@@ -930,6 +930,29 @@ void addReplyHumanLongDouble(client *c, long double d) {
     }
 }
 
+#define ADD_REPLY_LONG_LONG_WITH_PREFIX(c, ll, prefix, shared_hdr) do {       \
```

Comment:
why not using inline function instead macro?
it's hard to maintain too long macro.

Reply:
addressed in the latest commit :)

Reply:
https://github.com/redis/redis/pull/13520/commits/7f3717aead849b830043feb593f0b62e874f01d4